### PR TITLE
[WIP] Fix Transcription Errors

### PR DESCRIPTION
**Pull request by wopian**
Please help review changes made to files marked as completed. Thanks :smile:

Edit: a lot of these changes are stripping trailing white space which can be safely ignored while checking for any errors I've made.

---

Progress:
1. AUTOMATIC_MANEUVERS.agc (pages [1025](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1025.jpg) to [1036](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1036.jpg))
   - [x] Completed
   - [x] Changes checked
2. CM_BODY_ATTITUDE.agc (pages [0883](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0883.jpg) to [0889](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0889.jpg))
   - [x] Completed
   - [x] Changes checked
3. CM_ENTRY_DIGITAL_AUTOPILOT.agc (pages [1063](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1063.jpg) to [1092](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1092.jpg))
   - [ ] Completed
   - [x] Changes checked
4. CONIC_SUBROUTINES.agc (pages [1262](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1262.jpg) to [1308](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1308.jpg))
   - [ ] Completed
   - [ ] Changes checked
5. CONTRACT_AND_APPROVALS.agc (page [0001](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0001.jpg))
   - [x] Completed
   - [x] Changes checked

### CODE REVIEW

**Comment by oldmud0** [comment]
Woops, I didn't know checking a box edits the PR automatically!

**Comment by wopian** [comment]
Jeez, CONIC is nearly 2000 lines 😭

File: Comanche055/AUTOMATIC_MANEUVERS.agc [hunk 1]
```diff
@@ -182,10 +182,10 @@ ENDDAMP		TS	HOLDFLAG	# SET HOLDFLAG +0
 #
 # DETERMINE THE LOCATION OF THE RATE ERROR AND THE ATTITUDE ERROR RELATIVE TO THE SWITCHING LOGIC IN THE PHASE
 # PLANE.
-#
-# COMPUTE THE CHANGE IN RATE CORRESPONDING TO THE ATTITUDE ERROR NECESSARY TO DRIVE THE S/C INTO THE
+# COMPUTE THE CHANGE IN RATE CORRESPONDING TO THE ATTITUDE ERROR NECESSARY TO DRIVE THE THE S/C INTO THE
```
Comment (Philekunf) [inline Comanche055/AUTOMATIC_MANEUVERS.agc]: .

Comment (wopian) [inline Comanche055/AUTOMATIC_MANEUVERS.agc]: Hmm? [Page 1029 source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1029.jpg)

**Comment by wopian** [comment]
Both of the final two files are almost done (albeit one has no commits yet), however I'm now on another holiday I wasn't sure I was going on until yesterday.

So this won't be completed and merged for another 6-7 days 😥

**Comment by oldmud0** [comment]
Sorry, I am "too busy" "preparing" for school thanks to all the assignments right before I left.

**Comment by chrislgarry** [comment]
No worries guys.

File: Comanche055/CM_ENTRY_DIGITAL_AUTOPILOT.agc [hunk 2]
```diff
@@ -992,33 +992,32 @@ TIMSCAL		TS	TOFF		# IN CS
...
 		TS	TON2		# IN CS
 
-		CA	ZERO		# CANNOT REDO AFTER TIMETST.  TUSED GONE
+		CA	ZERO		# CANNOT REDO AFTER TIMETST. TUSED GONE
 		TS	JETAG		# SET +0 TO SHOW ROLL DAP CALLED.
 
-					# CAUSE THE TM OF BODY RATES VIA UPBUFF TO BE
-					# INITIALIZED.  ALSO CAUSE NEEDLES TO BE DONE ON EXIT
-					# AND ON ALTERNATE PASSES THROUGH CM/DUMPR.
+			     # CAUSE THE TM OF BODY RATES VIA UPBUFF TO BE
```
Comment (oldmud0) [inline Comanche055/CM_ENTRY_DIGITAL_AUTOPILOT.agc]: Might as well be tabs.

Comment (wopian) [inline Comanche055/CM_ENTRY_DIGITAL_AUTOPILOT.agc]: It's half the width of a single tab.

On another note, you've made me notice I've accidentally hit the shortcut to convert all the tabs in this file into spaces...so i'm gonna have to redo that commit.

Comment (oldmud0) [inline Comanche055/CM_ENTRY_DIGITAL_AUTOPILOT.agc]: Half-tabs?

File: Comanche055/CONTRACT_AND_APPROVALS.agc [hunk 3]
```diff
@@ -14,60 +14,59 @@
...
-#	INSTRUMENTATION LABORATORY, MASSACHUSETTS INSTITUTE OF		*
-#	TECHNOLOGY, CAMBRIDGE, MASS.					*
-#									*
-#************************************************************************
+# *************************************************************************
+# *									  *
+# *	     THIS AGC PROGRAM SHALL ALSO BE REFERRED TO AS:		  *
+# *									  *
+# *									  *
+# *		            COLOSSUS 2A				          *
```
Comment (oldmud0) [inline Comanche055/CONTRACT_AND_APPROVALS.agc]: There is a random tab wedged in here.

File: Comanche055/CONTRACT_AND_APPROVALS.agc [hunk 4]
```diff
@@ -14,60 +14,59 @@
...
 
-#	APPROVED:	DANIEL J. LICKLY		DATE:	28 MAR 69
+#	    APPROVED:	DANIEL J. LICKLY		   DATE:  28 MAR 69
 #		D.J.LICKLY, DIRECTOR, MISSION PROGRAM DEVELOPMENT
 #		APOLLO GUIDANCE AND NAVIGATION PROGRAM
 
-#	APPROVED:	FRED H. MARTIN			DATE:	28 MAR 69
-#		FRED H. MARTIN, COLOSSUS PROJECT MANAGER
+#	    APPROVED: FRED H. MARTIN			   DATE:  28 MAR 69
+#		FRED H. MARTIN, COLOSSUS PROJECT MANGER
```
Comment (oldmud0) [inline Comanche055/CONTRACT_AND_APPROVALS.agc]: Oh my gosh... of all the things that could have been screwed up... page 1?? Seriously?

File: Comanche055/CONTRACT_AND_APPROVALS.agc (hunk 3 above)
Comment (wopian) [inline Comanche055/CONTRACT_AND_APPROVALS.agc]: Patch should fix this, also added a space for the first line in this box which I never noticed on both checks of this file.

File: Comanche055/AUTOMATIC_MANEUVERS.agc [hunk 5]
```diff
@@ -433,34 +437,35 @@ J24		CS	AERR
...
+#	      T  = J/M(DELTA W)
+#	       J
 #
 #	DELTA W = DESIRED CHANGE IN S/C ANGULAR RATE AS DETERMINED BY THE
-#		  SWITCHING LOGIC, AT THIS POINT STORED IN KMPAC.
+#		  SWITCHING LOGIC,  AT THIS POINT STORED IN KMPAC.
 #
-#	    J/M = S/C INERTIA TO TORQUE RATIO SCALED BY
-#		  	(57.3/450)(B24/1600)(1/.8)
+#	   J/M	= S/C INERTIA TO TORQUE 9ATIO SCALED BY
```
Comment (astrofrog) [inline Comanche055/AUTOMATIC_MANEUVERS.agc]: This change doesn't seem right? RATIO is correct?

File: Comanche055/CM_BODY_ATTITUDE.agc [hunk 6]
```diff
@@ -190,46 +190,46 @@ CM/TRIO		STODL	24D
...
 		BZMF	+3		# SET GAMDOT=+0 AS TAG IF TOO SMALL.
-		
-NOGAMDOT	CA	ZERO		# COME HERE INHINTED
+
+NOGAMDOT	CA	ZERO		# COME HERE INHINTED.
 		TS	GAMDOT
-					# FOR NOW LEAVE IN 2'S C
-					# UPDATE ANGLES BY CORRECTING EULER ANG
+					# FOR NOW LEAVE IN 2S.C
+					# UPDATE ANGLES BY CORRECTING EUILER ANG
```
Comment (rogersachan) [inline Comanche055/CM_BODY_ATTITUDE.agc]: Are you sure it isn't Euler?

Comment (haisamido) [inline Comanche055/CM_BODY_ATTITUDE.agc]: That should Euler https://en.wikipedia.org/wiki/Euler_angles

**Comment by wopian** [comment]
Due to this PR being in such a bad state, with it in several states of review (with several sections just plain wrong) and some sections incomplete. It's become difficult to get back into checking my revisions for errors.

So i'll be closing this and re-doing it from scratch, with only 1 file at a time.

**Pull request closed without merging**